src/components/game/index.jsx

```diff
@@ -77,15 +78,19 @@ class GameScreen extends React.Component {
     this.refs.controls.update();
     this.refs.stats.forceUpdate();
 
-    const cameraRotated = this.prevCameraRotation === null ||
-      !this.prevCameraRotation.equals(this.camera.quaternion);
+    const cameraMoved =
+      this.prevCameraRotation === null ||
+      this.prevCameraPosition === null ||
+      !this.prevCameraRotation.equals(this.camera.quaternion) ||
+      !this.prevCameraPosition.equals(this.camera.position);
 
-    session.world.animate(this.clock.getDelta(), this.camera, cameraRotated);
+    session.world.animate(this.clock.getDelta(), this.camera, cameraMoved);
```

Comment:
Good idea!

Let's tackle both the name and locating it within the camera object in a separate PR.